src/Command/FindPoints.php

```diff
@@ -27,7 +30,7 @@ public function findPoints(FindPointsRequest $request): FindPointsResponse
     {
         $httpRequest = new HttpRequest(
             $request->getMethod(),
-            new Uri(ConfigProvider::API_VERSION . $request->getRequestUrl()),
+            new Uri($this->configProvider->apiVersion . $request->getRequestUrl()),
```

Comment:
Refers to the comment above :point_up: Then the API version does not need to be appended to every request. Am I right?